openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -454,7 +454,7 @@ def test_patch_account_unknown_user(self, api_client, user):
         ("country", "GB", "XY", u'"XY" is not a valid choice.'),
         ("year_of_birth", 2009, "not_an_int", u"A valid integer is required."),
         ("name", "bob", "z" * 256, u"Ensure this value has at most 255 characters (it has 256)."),
-        ("name", u"ȻħȺɍłɇs", "z   ", "The name field must be at least 2 characters long."),
+        ("name", u"ȻħȺɍłɇs", "z   ", u"The name field must be at least 2 characters long."),
```

Comment:
Made these test strings unicode strings - and removed a couple of `OrderedDict`s in order to make tests pass.